ui/components/GrafanaCustomChart.js

```diff
@@ -425,7 +425,7 @@ class GrafanaCustomChart extends Component {
       const processReceivedData = (result) => {
         self.props.updateProgress({ showProgress: false });
         if (typeof result !== 'undefined') {
-          const fullData = self.transformDataForChart(result, target);
```

Comment:
Why remove the target?

Reply:
It was not getting used, in the `transformDataForChart` I don't think increasing an argument is better.